dev.exs

```diff
@@ -360,6 +1338,203 @@ seeds = fn ->
+    thumbnail: "https://static.shuffle.dev/components/preview/43b384c1-17c4-470b-8332-d9dbb5ee99d7/footers/01_1648bd354f.png",
+    blueprint: ComponentDefinitionHelpers.default_definition(),
+    component_category_id: footers.id
+  })
+
+  Beacon.ComponentDefinitions.create_component_definition!(%{
+    name: "Footer 1",
+    thumbnail: "https://static.shuffle.dev/components/preview/43b384c1-17c4-470b-8332-d9dbb5ee99d7/footers/01_1648bd354f.png",
+    blueprint: ComponentDefinitionHelpers.default_definition(),
+    component_category_id: stats.id
+  })
+
+  Beacon.ComponentDefinitions.create_component_definition!(%{
+    name: "Footer 2",
+    thumbnail: "https://static.shuffle.dev/components/preview/43b384c1-17c4-470b-8332-d9dbb5ee99d7/footers/01_1648bd354f.png",
+    blueprint: ComponentDefinitionHelpers.default_definition(),
+    component_category_id: stats.id
+  })
+
+  Beacon.ComponentDefinitions.create_component_definition!(%{
+    name: "Footer 3",
+    thumbnail: "https://static.shuffle.dev/components/preview/43b384c1-17c4-470b-8332-d9dbb5ee99d7/footers/01_1648bd354f.png",
+    blueprint: ComponentDefinitionHelpers.default_definition(),
+    component_category_id: stats.id
+  })
+
+  Beacon.ComponentDefinitions.create_component_definition!(%{
+    name: "Sign In 1",
+    thumbnail: "https://static.shuffle.dev/components/preview/43b384c1-17c4-470b-8332-d9dbb5ee99d7/sign-in/01_b25eff87e3.png",
+    blueprint: ComponentDefinitionHelpers.default_definition(),
+    component_category_id: sign_ins.id
+  })
+
+  Beacon.ComponentDefinitions.create_component_definition!(%{
+    name: "Sign In 2",
+    thumbnail: "https://static.shuffle.dev/components/preview/43b384c1-17c4-470b-8332-d9dbb5ee99d7/sign-in/01_b25eff87e3.png",
+    blueprint: ComponentDefinitionHelpers.default_definition(),
+    component_category_id: sign_ins.id
+  })
+
+  Beacon.ComponentDefinitions.create_component_definition!(%{
+    name: "Sign In 3",
+    thumbnail: "https://static.shuffle.dev/components/preview/43b384c1-17c4-470b-8332-d9dbb5ee99d7/sign-in/01_b25eff87e3.png",
+    blueprint: ComponentDefinitionHelpers.default_definition(),
+    component_category_id: sign_ins.id
+  })
+
+  Beacon.ComponentDefinitions.create_component_definition!(%{
+    name: "Title",
+    thumbnail: "/component_thumbnails/title.jpg",
+    blueprint: ComponentDefinitionHelpers.create_definition("header", %{}, ["I'm a sample title"]),
+    component_category_id: basics.id
+  })
+
+  Beacon.ComponentDefinitions.create_component_definition!(%{
+    name: "Button",
+    thumbnail: "/component_thumbnails/button.jpg",
+    blueprint: ComponentDefinitionHelpers.create_definition("button", %{}, ["I'm a sample button"]),
+    component_category_id: basics.id
+  })
+
+  Beacon.ComponentDefinitions.create_component_definition!(%{
+    name: "Link",
+    thumbnail: "/component_thumbnails/link.jpg",
+    blueprint: ComponentDefinitionHelpers.create_definition("a", %{href: "#"}, ["I'm a sample link"]),
```

Comment:
Since we are aiming to parse HEEx templates we don't need to store the intermediary data structure anymore, instead this should be a HEEx template string:

```suggestion
    blueprint: """
    <a href="#">I'm a sample link</a>
    """
```

We'll need to implement the HEEx parser to generate the JSON that Page Builder expects.

Reply:
HEEx support to be added in a following PR based on https://github.com/BeaconCMS/beacon/issues/293